iXBRLViewerPlugin/__init__.py

```diff
@@ -115,6 +115,8 @@ def generateViewer(
     :param features: List of feature names to enable via generated JSON data.
     """
     # extend XBRL-loaded run processing for this option
+    if saveViewerDest is None:
+        return
```

Comment:
This is fine, but it leaves me wondering what the later `if out:` is doing.  From the history, we previously had:

```
out = getattr(options, 'saveViewerFile') or kwargs.get("responseZipStream")
```

But that got removed, so I suspect the `if out` is now redundant.

Reply:
Updated